hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/ContainerBalancerStatusSubcommand.java

```diff
@@ -60,12 +64,15 @@ public void execute(ScmClient scmClient) throws IOException {
     boolean isRunning = response.getIsRunning();
     ContainerBalancerStatusInfo balancerStatusInfo = response.getContainerBalancerStatusInfo();
     if (isRunning) {
+      Instant startedAtInstant = Instant.ofEpochSecond(balancerStatusInfo.getStartedAt());
       LocalDateTime dateTime =
-          LocalDateTime.ofInstant(Instant.ofEpochSecond(balancerStatusInfo.getStartedAt()), ZoneId.systemDefault());
+          LocalDateTime.ofInstant(startedAtInstant, ZoneId.systemDefault());
       System.out.println("ContainerBalancer is Running.");
 
       if (verbose) {
-        System.out.printf("Started at: %s %s%n%n", dateTime.toLocalDate(), dateTime.toLocalTime());
+        System.out.printf("Started at: %s %s%n", dateTime.toLocalDate(), dateTime.toLocalTime());
+        Duration balancingDuration = Duration.between(startedAtInstant, OffsetDateTime.now());
+        System.out.printf("Balancing duration: %s%n%n", getPrettyDuration(balancingDuration));
```

Comment:
Why not use the `toString` method of `Duration` instead of writing our own?

Reply:
I think '1h 30m' better than 'PT1H30M'.